src/pages/profiles/[...geoIdOrCountrySlug].js

```diff
@@ -4,21 +4,21 @@ import PropTypes from 'prop-types';
 import Head from 'next/head';
 import dynamic from 'next/dynamic';
 
-import config from '../../../config';
+import config from '../../config';
 
-import getProfileProps from '../../../components/ProfilePage/getInitialProps';
-import getAnalysisProps from '../../../components/AnalysisPage/getInitialProps';
+import getProfileProps from '../../components/ProfilePage/getInitialProps';
+import getAnalysisProps from '../../components/AnalysisPage/getInitialProps';
 
 const Analysis = dynamic({
   ssr: true,
-  loader: () => import('../../../components/AnalysisPage')
+  loader: () => import('../../components/AnalysisPage')
 });
 const Profile = dynamic({
   ssr: true,
-  loader: () => import('../../../components/ProfilePage')
+  loader: () => import('../../components/ProfilePage')
 });
 
-function ProfileOrAnalysisPage({ isAnalysis, initialProps }) {
+function ProfileOrAnalysisPage({ isProfile, initialProps }) {
```

Comment:
😆 ?

Reply:
I wish... had to reverse from _if not analysis, assume profile_ to _if not profile, assume analysis_ to support dynamic profiles.